backend/app/views/spree/admin/return_authorizations/_return_authorization_breadcrumb.html.erb

```diff
@@ -0,0 +1 @@
+<%= text %> (<%= t(state, scope: 'spree.return_authorization_states') %>)
```

Comment:
This is an interesting one. This particular breadcrumb could be perfectly specified from the controller itself using simple string interpolation. However,  In fact, I did it like that in an initial version. However, the string interpolated ended up being a bit too long and a bit too complex and I, since I had to implement the ability to use custom partials for another breadcrumb, I thought it would make sense to use it in this case for clarity.

Now, one thing I wondered is whether this additional level of complexity (i.e. the ability to specify custom breadcrumbs with partials) was really justified if we only needed for one or two breadcrumbs. Everything would be simpler if all the breadcrumbs would be just text, and didn't need special HTML formatting like in [this particular case][1] in the whole app. I'd appreciate your thoughts on this. Is it really needed and valuable to show the status information with colours in the breadcrumbs?

[1]:https://github.com/solidusio/solidus/pull/3191/files#diff-e249441f4fa5641286d474939a6db2f3

Reply:
I think we can remove the HTML from breadcrumbs, even if it probably requires a new PR and an alternative UI method to show the state of the parent object in the page, which I think was the original reason why they have been introduced.